app/models/workers_scaling/experiment_statistics_factory.rb

```diff
@@ -0,0 +1,19 @@
+module WorkersScaling
+  class ExperimentStatisticsFactory
+    @cache = ActiveSupport::Cache::MemoryStore.new expires_in: 10.minutes
```

Comment:
Oba pytania do @kliput - widziałem, że jest tam dostępne, więc używałem. U siebie też stworzę osobny cache, tak będzie lepiej - w razie zmiany w całym systemie tutaj zostanie to co chcemy.